src/vs/platform/accessibility/browser/accessibilityService.ts

```diff
@@ -35,7 +35,7 @@ export class AccessibilityService extends Disposable implements IAccessibilitySe
 		this._register(this._configurationService.onDidChangeConfiguration(e => {
 			if (e.affectsConfiguration('editor.accessibilitySupport')) {
 				updateContextKey();
-				this._onDidChangeScreenReaderOptimized.fire();
+				this._onDidChangeScreenReaderOptimized.fire(this._configurationService.getValue('editor.accessibilitySupport'));
```

Comment:
isn't this setting an enum, not a boolean?

Reply:
oh yes 👍🏼